public/app/engagementindex/leaderboard/leaderboardController.js

```diff
@@ -114,7 +114,12 @@
               'backgroundColor': 'transparent',
               'inverted': true,
               // Ensure that the box plot is displayed horizontally
-              'margin': [0, 20, 0, 20],
+              'margin': [
+                0,
+                20,
+                0,
+                20
+              ],
```

Comment:
The rules I've put in require multi-line arrays when length > 3. In most cases, it's reasonable and good. In a few cases (as above), it's silly. IMO, benefits outweigh the silliness.